src/components/i-personal-data.ts

```diff
@@ -0,0 +1,10 @@
+export interface IPersonalDataForm {
```

Comment:
I would rather stick to one convention while naming files. You are using kebab-case here while all components seems to be capitalized.

Reply:
renamed